tests/callbacks/test_callbacks.py

```diff
@@ -214,18 +200,18 @@ def on_test_end(self, trainer, pl_module):
 
 def test_early_stopping_no_val_step(tmpdir):
     """Test that early stopping callback falls back to training metrics when no validation defined."""
-    class ModelWithoutValStep(LightTrainDataloader, TestModelBase):
 
+    class CurrentModel(EvalModelTemplate):
         def training_step(self, *args, **kwargs):
             output = super().training_step(*args, **kwargs)
-            loss = output['loss']  # could be anything else
-            output.update({'my_train_metric': loss})
+            output.update({'my_train_metric': output['loss']})  # could be anything else
             return output
 
-    model = ModelWithoutValStep(tutils.get_default_hparams())
+    model = CurrentModel(tutils.get_default_hparams())
+    model.validation_step = LightningModule.validation_step
```

Comment:
Again, no need to use LightningModule...

Reply:
we found it cleaner to see that this method was not overwritten... so you want to have it?
```
model.validation_step = None
```